learn/guides/ballerina-shell.md

```diff
@@ -31,23 +34,39 @@ Type /exit to exit and /help to list available commands.
 =$ 
 ```
 
-## Stopping the Shell
+## Available commands
 
-Execute the `/exit` command to exit the Shell.
+Execute the `/help` command to view available commands.
 
 ```bash
-=$ /exit                                                                                                                                                                                                           
-| Bye!!!
+=$ /help                 
+| Ballerina Shell Help
+| 
+| Type a Ballerina language expression, statement, or declaration.
+| Or type one of the following commands:
+| 
+| /exit           - Exit the shell.
+| /debug          - Toggle the debug mode ON/OFF.
+| /reset          - Reset REPL state.
```

Comment:
```suggestion
| /reset          - Reset the REPL state.
```

Reply:
Updated in 2a10777065a5455f4155b3fbdd75b7a70f169c75